src/components/DateRangePicker.jsx

```diff
@@ -51,6 +51,7 @@ const defaultProps = {
   screenReaderInputMessage: '',
   showClearDates: false,
   showDefaultInputIcon: false,
+  inputIconPosition: 'before',
```

Comment:
Now that we have the constants file, it'd be better to import that and use `BEFORE_POSITION` here.